src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -480,6 +480,8 @@ OperatorsBridge::OperatorsBridge() {
     REGISTER_OPERATOR("RandomUniformLike", 1, random_uniform_like);
     REGISTER_OPERATOR("Reciprocal", 1, reciprocal);
     REGISTER_OPERATOR("ReduceLogSum", 1, reduce_log_sum);
+    register_operator("ReduceLogSum", VersionRange{1, 17}, op::set_1::reduce_log_sum);
```

Comment:
Great, thanks to use modern style! :) Do not forget to add opset-13 :)

Reply:
Doesn’t it already include 13? After a talk with @mitruska we decided to keep it this way since there are no differences in the subgraphs generated.